### ISSUE TITLE
Getting 40-50 email notifications for each comment made

### ISSUE THREAD

**Issue by Kevin-at-OOYR** [stcr/subscribe-to-comments-reloaded#672]
My subscription is sending out duplicate notifications of each comment, often 50 or more notifications per comment. How do I stop this and limit the notifications to just one per recipient? 

Site is https://outofyourrut.com

**Comment by Kevin-at-OOYR** [comment]
Apparently it's just me. I checked with one subscriber and she said she only gets one.
________________________________

**Comment by BobaWebDev** [comment]
If you have the option "BBC admin on notifications" enabled you will receive a copy of every notification sent.  Is that option enabled in WP admin > STCR > Options?

**Comment by Kevin-at-OOYR** [comment]
Yes.
________________________________

**Comment by BobaWebDev** [comment]
Are there 50+ subscribers for the comments on that post?

**Comment by Kevin-at-OOYR** [comment]
Yes, but the multiple email notifications are happening on all comments, even those with very few subs.

________________________________

**Comment by BobaWebDev** [comment]
Can you install [WP Mail Logging](https://wordpress.org/plugins/wp-mail-logging/) plugin?

It will log all the emails being sent out.

When you receive the duplicate emails can you check the log and see if there are emails in the log that are sent to the same email address multiple times.

The "BBC admin on notifications" sends you a copy of the email but it's under the same "email request", so if you are receiving duplicates then the subscriber must receive them as well. 

Let me know what results come from that test so we know where to look.

**Comment by Kevin-at-OOYR** [comment]
I just downloaded it. I'll need to wait for a new comment to come it to check the log, if that's correct?
________________________________

**Comment by BobaWebDev** [comment]
Yes, the logging starts after plugin activation. You can make a comment yourself to trigger the notifications, but your subscribers would receive the notifications as well, so maybe best to wait for a real comment.

**Comment by BobaWebDev** [comment]
When you click on those items in the log file do they have the same receiver?

**Comment by Kevin-at-OOYR** [comment]
Where am I looking for the log file?
________________________________

**Comment by BobaWebDev** [comment]
WP admin > Tools > WP Mail Log.  Every email sent out is logged in there, find the part where you got the 51 email notifications (based on the times) and you can click "View" button to see more info, in that info at the top you'll see "Receiver". That's the email address it was sent to. Check if the same receiver is shown multiple times.

**Comment by Kevin-at-OOYR** [comment]
Let me check...
________________________________

**Comment by Kevin-at-OOYR** [comment]
It's showing one each. Now I turned off my notices yesterday, but only after two or three notification rounds. I only appear on the list once. But three of the comments generated the 50+ responses each before I turned off my notifications. Only one notification to me is showing out of the 150+ I received.
________________________________

**Comment by BobaWebDev** [comment]
You're BCC on those emails from the "BCC admin" feature, not the primary
receiver, you won't show as the receiver in the logs.

Check if the receivers repeat, if they don't repeat then it's fine, you're
just getting a copy of the emails the subscribers get.

That's the option mentioned previously. Turning it off will disable that
functionality.

**Comment by Kevin-at-OOYR** [comment]
The other receivers got one only. I did disable notifications yesterday. Is there any way to turn the notifications on without the duplicates?
________________________________

**Comment by BobaWebDev** [comment]
The "BBC admin on notifications" is for the purpose for you to receive duplicate/copy of every notification. So if you don't want that, keep it disabed.

If you just want to receive a single notification when someone comments then enable the "Subscribe authors" option. It automatically subscribes the author of the post. It's not retroactive, so if you enable it now, you will be automatically subscribed to the future posts you publish.

There's also the "Let admin subscribe" option, setting that to "Yes", will allow you to manually subscribe to comments on a post, same way your visitors do.

**Comment by Kevin-at-OOYR** [comment]
Thanks Slobodan, I've made the change and I'll let you know if it works. With the holiday in the US, I don't know if there will be any comments for a couple of days.

Kevin
________________________________

**Comment by Kevin-at-OOYR** [comment]
Also Slobodan, you're very good at what you do, which I'm very thankful for.
________________________________

**Comment by BobaWebDev** [comment]
You're welcome Kevin, and thank you.

**Comment by Kevin-at-OOYR** [comment]
Hi Slobodan - That seems to have corrected the problem. Now I'm getting only one notification per comment.

Thanks again,

Kevin
________________________________

**Comment by BobaWebDev** [comment]
Great, happy to hear that.

You're welcome.
